### ISSUE TITLE
Abnormal broncho-alveolar lavage fluid morphology

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5544]
**Preferred term label:**
Abnormal broncho-alveolar lavage fluid morphology

**Preferred term label:**
Abnormal broncho-alveolar lavage fluid morphology
**Synonyms**
Abnormality in broncho-alveolar lavage fluid
**Definition (free text, please give PubMed ID)**
see https://pubmed.ncbi.nlm.nih.gov/30173880/

**Parent term (use hpo.jax.org/app)**
Abnormal lung morphology 
https://hpo.jax.org/app/browse/term/HP:0002088

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
-- depends on sub term

**Your nano-attribution (ORCID)**

@mgriese
@GrieseAG

**Comment by pnrobinson** [comment]
duplicated #5543
